model/versioning/Versioned.php

```diff
@@ -202,10 +204,10 @@ public static function reset() {
 	 * Construct a new Versioned object.
 	 *
 	 * @var array $stages The different stages the versioned object can be.
-	 * The first stage is considered the 'default' stage, the last stage is
-	 * considered the 'live' stage.
+	 * The first stage must be the value of self::DRAFT_STAGE. If more than one stage is given,
+	 * the final stage must be the value of self::LIVE_STAGE.
 	 */
-	public function __construct($stages = array('Stage','Live')) {
+	public function __construct($stages = array(self::DRAFT_STAGE, self::LIVE_STAGE)) {
```

Comment:
I had that thought too, but you can pass 1 stage or 3, technically, just the first and last are important